### PR TITLE
RDoc-1122 Client API > operations > attachments

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+## Syntax
+
+{CODE:csharp get_attachment_syntax@ClientApi\Operations\Attachments.cs /}
+
+{CODE:csharp get_attachment_return_value@ClientApi\Operations\Attachments.cs /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **documentId** | string | Name of a document which will contain attachment |
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown]: Name -> ID

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown]: All other places: document name -> document id

File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Attachments.cs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        }
+
+
+        public Attachments()
+        {
+            using (var store = new DocumentStore())
+            {
+                {
+                    #region delete_1
+                    store.Operations.Send(new DeleteAttachmentOperation("orders/1", "invoice.pdf"));
```
Comment (arekpalinski) [inline Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Attachments.cs]: orders/1 -> orders/1-A in all examples

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown (hunk 1 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown]: done

File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Attachments.cs (hunk 2 above)
Comment (ml054) [reply Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Attachments.cs]: done

**Comment by ml054** [comment]
I've pushed the changes. @arekpalinski Please review.

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/delete-attachment.dotnet.markdown [hunk 3]
```diff
@@ -1 +1,23 @@
...
+
+This operation is used to delete attachment from document. 
+
+## Syntax
+
+{CODE:csharp delete_attachment_syntax@ClientApi\Operations\Attachments.cs /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **documentId** | string | Id of a document containing attachment |
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/delete-attachment.dotnet.markdown]: Id -> ID (in description)

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown [hunk 4]
```diff
@@ -1 +1,37 @@
...
+
+## Syntax
+
+{CODE:csharp get_attachment_syntax@ClientApi\Operations\Attachments.cs /}
+
+{CODE:csharp get_attachment_return_value@ClientApi\Operations\Attachments.cs /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **documentId** | string | Id of a document which will contain attachment |
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/get-attachment.dotnet.markdown]: Id -> ID (in description)

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/put-attachment.dotnet.markdown [hunk 5]
```diff
@@ -1 +1,36 @@
...
+
+## Syntax
+
+{CODE:csharp put_attachment_syntax@ClientApi\Operations\Attachments.cs /}
+
+{CODE:csharp put_attachment_return_value@ClientApi\Operations\Attachments.cs /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **documentId** | string | Id of a document which will contain attachment |
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/attachments/put-attachment.dotnet.markdown]: Id -> ID (in description)

**Comment by ml054** [comment]
Fixed. Please review.

**Pull request merged**